dotnet/samples/ApplicationInsightsExample/Program.cs

```diff
@@ -130,7 +130,7 @@ private static IKernel GetKernel(ILogger logger)
         ILogger logger,
         int maxTokens = 1024)
     {
-        var plannerConfig = new SequentialPlannerConfig { MaxTokens = maxTokens };
+        var plannerConfig = new SequentialPlannerConfig { MaxTokens = maxTokens, Memory = kernel.Memory };
```

Comment:
I don't think this necessarily needed since we aren't looking for functions saved in memory here.  Your call.